web/ui/static/css/main.css

```diff
@@ -1790,7 +1787,7 @@ input[type=file] {
 .jelly-table .loader{
     opacity: 1;
     width: 100%;
-    height: calc(100% - 59px);
+    height: 80%;
```

Comment:
height of 80% is not enough in the case of taller tables like services. The calculated height is much closer to correct, though the hardcoded offset is not completely accurate since the table headers can sometimes be taller than 59px.